Add learning goals for Mobility tutorials (#2883)
viamrobotics/docs: docs/tutorials/services/navigate-with-rover-base.md

# SMEs: Ray Bjorkman, Fahmina
---

> 

Author: ```suggestion
```

> Using the navigation service, you can queue up user-defined waypoints and your machine will move to them in the order that you specify.

Reviewer: ```suggestion
Using the [navigation service](/mobility/navigation/), you can queue up user-defined waypoints and your machine will move to them in the order that you specify.
```

Author: I can also do that - not sure if it duplicates the rest in the intro too much?

Reviewer: You mean since it makes the sentence more similar to "add waypoints to specify locations you want your machine to move to, both in the app and programmatically" (line 48)? I think this is still better because the gerund as the first word in the doc just reads oddly to me. Up to you.

# SMEs: Ray Bjorkman, Fahmina
---

> The [navigation service](/mobility/navigation/) is the stateful definition of the [motion service](/mobility/motion/).

Reviewer: [opt] Since many users don't know what "stateful definition" means, could change, but that'd be out of diff

Author: I don't know what else to use - do you?

> The [navigation service](/mobility/navigation/) is the stateful definition of the [motion service](/mobility/motion/).

Author: ```suggestion
```
I think this can just go. The next sentences describe it

Reviewer: I see what you mean but I think it's awk as a first sentence--made a suggestion

> Using Navigation, you can queue up user-defined waypoints and your machine will move to them in the order that you specify.

Reviewer: ```suggestion
Using navigation, you can queue up user-defined waypoints and your machine will move to them in the order that you specify.
```
[nit]

> Using navigation, you can queue up user-defined waypoints and your machine will move to them in the order that you specify.

Author: ```suggestion
Using the navigation service, you can queue up user-defined waypoints and your machine will move to them in the order that you specify.
```

You can also add obstacles or set linear and angular velocity targets in your navigation service config.
Viam's motion planner will plan routes that avoid those obstacles and attempt to keep the robot at your specified velocity.

{{< alert title="Learning Goals" color="info" >}}

After following this tutorial, you will be able to:

> - add waypoints to specify locations you want your machine to move to both in the app and programmatically

Reviewer: ```suggestion
- add waypoints to specify locations you want your machine to move to, both in the app and programmatically
```
[nit] I think this is easier to parse

> - use the navigation service to start moving your machine along those waypoints

Reviewer: ```suggestion
- use the navigation service to move your machine along those waypoints
```
